src/pages/Welcome.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "styled-components";
+import MainBackground from "../components/MainBackground";
```

Comment:
You import MainBackground but I don't see it being used anywhere, is this on purpse ?

Reply:
It turned out to be useless component, I deleted it in the meantime.